### ISSUE TITLE
Mouse scroll/click not available in tmux on wsl (windows 10)

### ISSUE THREAD

**Issue by pszypowicz** [xtermjs/xterm.js#3019]
I think this is related to the https://github.com/xtermjs/xterm.js/issues/329

## Details
- Browser and browser version: Firefox 78
- OS version: Windows 10
- xterm.js version: 4.8.1

### Steps to reproduce

1. Run demo on windows
2. In shell open `wsl`
3. `tmux attach`
4. set -g mouse on
5. scroll does not work
6. split the window
7. changing windows with LMC does not work

**Comment by Tyriar** [comment]
It seems to work for me:

![recording (20)](https://user-images.githubusercontent.com/2193314/89207057-ff9fc100-d56e-11ea-853f-ec6d80ea17a7.gif)

What version of Windows 10 is this? Can you share your settings.json file?

**Comment by Tyriar** [comment]
I tried using wsl/tmux on Firefox and it gave the same results.

**Comment by pszypowicz** [comment]
I have insiders preview enabled (the build version is: 19041.388), so maybe that's the fault? I will try it on the laptop with stable release.

**Comment by Tyriar** [comment]
I'm on Microsoft Windows [Version 10.0.19041.421] 🤷

**Comment by pszypowicz** [comment]
Are you using the wsl2 version too?

**Comment by Tyriar** [comment]
Yes I'm on wsl2 as well:

```
$ cat ~/.tmux.conf
set -g mouse on

$ tmux -V
tmux 2.8
```

**Comment by pszypowicz** [comment]
On Ubuntu 20.04 I have

```
$tmux -V
tmux 3.0a
```

and on 18.04:
```
tmux 2.6
```

What distro are you using? 
By the way, thanks for helping!

**Comment by Tyriar** [comment]
I'm on Debian, which should act the same as Ubuntu.

**Comment by pszypowicz** [comment]
So I just installed debian image on both computers I own (one windows Insiders Preview and WSL2, and one Stable and wsl1). Neither worked. Also, debian was clean with only tmux installed + tmux.conf with mouse setting. Also it does not work for me in the VS Code, which is my goal (and the reason I'm so envy to see you screencast :D )

**Comment by Tyriar** [comment]
@pszypowicz what happens when you click? Does it do a regular selection?

What's in your settings.json file?

**Comment by pszypowicz** [comment]
Yes, it's just a regular selection, as I would not have mouse setting on in tmux at all. And if I try it directly in the {cmd, powershell, terminal} -> wsl -> tmux then everything works as expected. Just xterm does not work for me (and vs code as a result)
